content/docs/command-reference/commit.md

```diff
@@ -96,8 +96,8 @@ reproducibility in those cases.
 
 ## Examples
 
-Let's employ a simple <abbr>workspace</abbr> with some data, code, ML models,
-pipeline stages, such as the <abbr>DVC project</abbr> created for the
+Let's employ a <abbr>workspace</abbr> with some data, code, ML models, pipeline
+stages, such as the <abbr>DVC project</abbr> created for the
```

Comment:
I'm not sure this is condescending but no strong opinion, we can change it. However, I believe this same paragraph exists in other refs. such as in checkout so let's apply the change consistently everywhere please.

Reply:
@mformihir what about the use of "just" in this doc? See https://github.com/iterative/dvc.org/pull/1396/files#diff-aac9973cd919dc8d6735e0de82ab7f62L47-R47 Do you think it's condescending?

Reply:
> no strong opinion, we can change it

On this, also. I'm not sure the phrase "Let's employ a workspace with some data..." reads right, without the adjective. Let's change the word "simple" for something more specific such as "new" or "fresh" please.